lib/elixir/pages/Typespecs.md

```diff
@@ -35,9 +35,9 @@ The syntax Elixir provides for type specifications is similar to [the one in Erl
                                     ## Numbers
```

Comment:
I believe the examples are enough to clarify it. so let's get rid of all of them. :)